lib/modules/gasEstimator/index.js

```diff
@@ -51,7 +51,9 @@ class GasEstimator {
                 } else if (_.isEqual(variance[0], variance[1]) && _.isEqual(variance[1], variance[2])) {
                   gasMap[name] = variance[0];
                 } else {
-                  gasMap[name] = 'infinite';
+                  let sum = _.reduce(variance, function(memo, num) { return memo + num; }, 0);
+                  let average = sum / variance.length;
+                  gasMap[name] = average;
```

Comment:
You could get rid of the average variable: `gasMap[name] = sum / variance.length;`